maintenance/doMaintenance.php

```diff
@@ -124,14 +126,23 @@
 // Much much faster startup than creating a title object
 $wgTitle = null;
 
+\Wikia\Logger\WikiaLogger::instance()->info("Maintenance script $maintClass started.");
```

Comment:
`$maintClass` is logged as `maintenance_class` context field. Should duplicate this information where?